src/client/components/Navigation/NavAssessment/Placeholder.tsx

```diff
@@ -0,0 +1,61 @@
+// NOTE: This file is a placeholder and will be removed in next PR.
```

Comment:
This file can be safely ignored, it will be removed in next pull request
It is used to show content for the video

We show actual data here, which is neat 
Worked quite well although the UI is malformed a bit because we use existing component (`RecentActivityItem`)